src/controllers/bookings.cr

```diff
@@ -3,7 +3,7 @@ class Bookings < Application
 
   before_action :find_booking, only: [:show, :update, :update_alt, :destroy, :check_in, :approve, :reject]
   before_action :check_access, only: [:update, :update_alt, :destroy, :check_in]
-  getter booking : Booking?
+  getter(booking : Booking?) { find_booking }
```

Comment:
The type should be `Booking` as the return type of the method in the block will always be a concrete value
```suggestion
  getter booking : Booking { find_booking }
```